plugins/github-issues/api-report.md

```diff
@@ -27,7 +27,41 @@ export const gitHubIssuesPlugin: BackstagePlugin<
 export type GitHubIssuesProps = {
   itemsPerPage?: number;
   itemsPerRepo?: number;
+  filterBy?: IssuesFilters;
```

Comment:
Let's prefix these new types with `Github` to make exports a bit more contained to this plugin

That makes it `GithubIssuesFilters` and `GithubIssuesOrdering`